openmc/data/library.py

```diff
@@ -63,32 +63,29 @@ def register_file(self, filename):
         library = {'path': filename, 'type': filetype, 'materials': materials}
         self.libraries.append(library)
 
-    def export_to_xml(self, path='cross_sections.xml'):
+    def export_to_xml(self, path='cross_sections.xml', append=False):
         """Export cross section data library to an XML file.
 
         Parameters
         ----------
         path : str
             Path to file to write. Defaults to 'cross_sections.xml'.
+        append : bool
+            Whether to append to an existing file, if it exists.
+            Defaults to False.
 
         """
-        root = ET.Element('cross_sections')
 
-        # Determine common directory for library paths
-        common_dir = os.path.dirname(os.path.commonprefix(
-            [lib['path'] for lib in self.libraries]))
-        if common_dir == '':
-            common_dir = '.'
```

Comment:
The `openmc-get-photo-endf71` script places the photo-atomic hdf5 files in a separate directory and appends their location to an existing `cross_sections.xml` file if provided with the -c argument.  If `cross_sections.xml` has a `directory` element,  then that must be taken into account when setting the location of the photo-atomic hdf5 files.  I figured instead of checking for a `directory` element and making the new hdf5 locations relative to that, just set the full location of all the hdf5 files.